.github/scripts/generate-roadmap.js

```diff
@@ -0,0 +1,96 @@
+module.exports = async ({github, context}) => {  
+  // Configuration  
+  const ignoreLabels = ['bug', 'docs']; // Labels to ignore  
+  const roadmapLabel = 'roadmap'; // Special label that overrides ignoreLabels  
+  const fileName = "AUTO-ROADMAP.md"
```

Comment:
Since this just hoovers up issues, we'll need to groom them before this thing goes 'live'. In the meantime, we'll target a different file than `/ROADMAP.md`.